front/src/applications/operationalStudies/components/Scenario/ScenarioContent.tsx

```diff
@@ -256,15 +267,12 @@ const ScenarioContent = ({
         </div>
       </div>
       {/* TODO : add right column */}
-      <div
-        // style={{ display: displayConflictsList ? 'block' : 'none' }}
-        className="right-column"
-      >
+      <div style={{ display: isConflictsListDisplay ? 'block' : 'none' }} className="right-column">
         {conflicts && (
           <ConflictsList
             conflicts={conflicts}
-            timetableItems={[]}
-            onConflictClick={handleConflictClick}
+            timetableItems={filteredTimetableItems}
+            onConflictClick={() => handleConflictClick}
```

Comment:
I think that's not a valid `onConflictClick` callback? This is a function which takes no argument and returns a function.

Clicking on a conflict no longer selects the affected train.